package.json

```diff
@@ -23,16 +25,9 @@
       "react-app/jest"
     ]
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+  "browserslist": [
```

Comment:
This change was to appease the babel transpiler which was having issues with advanced syntax in the npm color package. Investigate how much of this browserlist config you can convert for your development build.